src/main/java/ted/Ui.java

```diff
@@ -0,0 +1,98 @@
+package ted;
+
+import java.util.Scanner;
+
+import ted.exception.TedException;
+import ted.task.Task;
+import ted.task.TaskList;
+
+/**
+ * Ui is use for terminal interaction with user
+ */
+public class Ui {
+
+    private static final String GREETING = "##################################################\n"
+            + "||                                              ||\n"
+            + "||                Hello! I'm Ted                ||\n"
+            + "||            What can I do for you?            ||\n"
+            + "||                                              ||\n"
+            + "##################################################\n";
+
+    private static final String INPUT_PREFIX = "> ";
+
+    /**
+     * Scanner to scan user input
```

Comment:
Small point but I think this comment is a bit self-explanatory and could be omitted

Reply:
I think it is required in our coding standard, will check it out!